pyproject.toml

```diff
@@ -9,6 +9,9 @@ python = "^3.7"
 flask = "^1.1.2"
 python-dotenv = "^0.14.0"
 requests = "^2.26.0"
+pytest = "^6.2.5"
+pytest-watch = "^4.2.0"
+selenium = "^4.1.0"
```

Comment:
These could go in the dev-dependencies section

(Would have been put there in the first place if you did `poetry add --dev ...`